### PR TITLE
Simplify mobile header

### PR DESCRIPTION
**Pull request by AlecRust**
Puts menu behind hamburger on mobile. Top bar is now fixed to the top at all screen widths.

**Note:** Adds a new "Toggle menu" translation, dunno if something else is required for this 🤷‍♂️ 

**Before (mobile):**
image

**After: (mobile)**
image

**Before: (desktop)**
image

**After: (desktop)**
image

### CODE REVIEW

File: web/templates/includes/panel.html [hunk 1]
```diff
@@ -102,25 +100,33 @@
...
               <a title="<?=_('Logs');?>" class="top-bar-nav-link <?php if($TAB == 'LOG') echo 'active' ?>" href="/list/log/">
                 <i class="fas fa-clock-rotate-left"></i>
-                <span class="u-hidden"><?=_('Logs');?></span>
+                <span class="top-bar-nav-link-label"><?=_('Logs');?></span>
               </a>
             </li>
           <?php } else { ?>
             <?php if ($panel[$user]['SUSPENDED'] === 'no') {?>
               <li class="top-bar-nav-item">
                 <a title="<?=htmlspecialchars($user)?> (<?=htmlspecialchars($panel[$user]['NAME'])?>)" class="top-bar-nav-link" href="/edit/user/?user=<?=$user; ?>&token=<?=$_SESSION['token']?>">
```
Comment (jaapmarcus) [inline web/templates/includes/panel.html:109]: Active is not "active" when you click on "User (xxxx") and the page is reloaded.

File: web/templates/includes/panel.html [hunk 2]
```diff
@@ -55,33 +74,12 @@
               <li class="top-bar-nav-item">
                 <a title="<?=_('Files');?>" class="top-bar-nav-link <?php if($TAB == 'FM') echo 'active' ?>" href="/fm/">
```
Comment (jaapmarcus) [inline web/templates/includes/panel.html]: Change it to "Filemanger" instead of "Files" as we have no more space.

**Review by jaapmarcus — CHANGES_REQUESTED** [review]

**Comment by jaapmarcus** [comment]
IMG_0253
IMG_0254

File: web/templates/includes/panel.html (hunk 1 above)
Comment (AlecRust) [reply web/templates/includes/panel.html:109]: Yeah, existing issue on 1.6 🤷

Comment (jaapmarcus) [reply web/templates/includes/panel.html:109]: It doesn't surprise me.

**Review by jaapmarcus — APPROVED** [review]

**Pull request merged**